src/app/devices/deviceEvents/components/deviceEvents.tsx

```diff
@@ -71,47 +70,51 @@ export const DeviceEvents: React.FC = () => {
     const { pnpState, } = usePnpStateContext();
     const modelDefinitionWithSource = pnpState.modelDefinitionWithSource.payload;
     const modelDefinition = modelDefinitionWithSource && modelDefinitionWithSource.modelDefinition;
-    const extendedModelDefinition = modelDefinitionWithSource && modelDefinitionWithSource.extendedModel;
     const isLoading = pnpState.modelDefinitionWithSource.synchronizationStatus === SynchronizationStatus.working;
-    const telemetrySchema = React.useMemo(() => getDeviceTelemetry(extendedModelDefinition || modelDefinition), [extendedModelDefinition, modelDefinition]);
+    const telemetrySchema = React.useMemo(() => getDeviceTelemetry(modelDefinition), [modelDefinition]);
     const [showPnpModeledEvents, setShowPnpModeledEvents] = React.useState(false);
 
     // simulation specific
     const [showSimulationPanel, setShowSimulationPanel] = React.useState(false);
 
-    React.useEffect(() => {
-        if (componentName) {
-            AppInsightsClient.getInstance()?.trackPageView({name: TELEMETRY_PAGE_NAMES.PNP_TELEMETRY});
-        } else {
-            AppInsightsClient.getInstance()?.trackPageView({name: TELEMETRY_PAGE_NAMES.DEVICE_TELEMETRY});
-        }
-    }, []); // tslint:disable-line: align
```

Comment:
Is it ok to leave this in?
It looks like we still have `componentName` populated so I think we should be good to leave the telemetry in?

Reply:
Thanks for catching it! I copied the old file from the older commits, and forgot what was newly added. If we remove it, there won't be any views on this page. Phew!